packages/sacado/test/UnitTests/CMakeLists.txt

```diff
@@ -377,6 +377,128 @@ IF (Sacado_ENABLE_TeuchosCore)
+
+      TRIBITS_ADD_EXECUTABLE_AND_TEST(
+        FadKokkosTests_HIP_Hierarchical
+        SOURCES Fad_KokkosTests.hpp
+                Fad_KokkosTests_HIP_Hierarchical.cpp
+        COMM serial mpi
+        STANDARD_PASS_OUTPUT
+        NUM_MPI_PROCS 1
+        RUN_SERIAL
+        )
+
+      TRIBITS_ADD_EXECUTABLE_AND_TEST(
+        FadKokkosTests_HIP_Hierarchical_SFad
+        SOURCES Fad_KokkosTests.hpp
+                Fad_KokkosTests_HIP_Hierarchical_SFad.cpp
+        COMM serial mpi
+        STANDARD_PASS_OUTPUT
+        NUM_MPI_PROCS 1
+        RUN_SERIAL
+        )
+
+      TRIBITS_ADD_EXECUTABLE_AND_TEST(
+        FadKokkosTests_Cuda_Hierarchical_DFad
+        SOURCES Fad_KokkosTests.hpp
+                Fad_KokkosTests_HIP_Hierarchical_DFad.cpp
+        COMM serial mpi
+        STANDARD_PASS_OUTPUT
+        NUM_MPI_PROCS 1
+        RUN_SERIAL
+        )
+
+      TRIBITS_ADD_EXECUTABLE_AND_TEST(
+        FadKokkosTests_NoViewSpec_HIP
+        SOURCES Fad_KokkosTests.hpp
+                Fad_KokkosTests_NoViewSpec_HIP.cpp
+        COMM serial mpi
+        STANDARD_PASS_OUTPUT
+        NUM_MPI_PROCS 1
+        RUN_SERIAL
+        )
+
+#      TRIBITS_ADD_EXECUTABLE_AND_TEST(
+#        FadFadKokkosTests_Cuda
+#        SOURCES Fad_Fad_KokkosTests.hpp
+#                Fad_Fad_KokkosTests_Cuda.cpp
+#        COMM serial mpi
+#        STANDARD_PASS_OUTPUT
+#        NUM_MPI_PROCS 1
+#        RUN_SERIAL
+#        )
+
+      TRIBITS_ADD_EXECUTABLE_AND_TEST(
+        FadKokkosAtomicTests_HIP
+        SOURCES Fad_KokkosAtomicTests.hpp
+                Fad_KokkosAtomicTests_HIP.cpp
+        COMM serial mpi
+        STANDARD_PASS_OUTPUT
+        NUM_MPI_PROCS 1
+        RUN_SERIAL
+        )
+
+      TRIBITS_ADD_EXECUTABLE_AND_TEST(
+        FadKokkosAtomicTests_NoViewSpec_HIP
+        SOURCES Fad_KokkosAtomicTests.hpp
+                Fad_KokkosAtomicTests_NoViewSpec_HIP.cpp
+        COMM serial mpi
+        STANDARD_PASS_OUTPUT
+        NUM_MPI_PROCS 1
+        RUN_SERIAL
+        )
+
+#      TRIBITS_ADD_EXECUTABLE_AND_TEST(
+#        FadKokkosAtomicTests_Cuda_Hierarchical
+#        SOURCES Fad_KokkosAtomicTests.hpp
+#                Fad_KokkosAtomicTests_Cuda_Hierarchical.cpp
+#        COMM serial mpi
+#        STANDARD_PASS_OUTPUT
+#        NUM_MPI_PROCS 1
+#        RUN_SERIAL
+#        )
+
+#      TRIBITS_ADD_EXECUTABLE_AND_TEST(
+#        FadKokkosAtomicTests_Cuda_Hierarchical_SFad
+#        SOURCES Fad_KokkosAtomicTests.hpp
+#                Fad_KokkosAtomicTests_Cuda_Hierarchical_SFad.cpp
+#        COMM serial mpi
+#        STANDARD_PASS_OUTPUT
+#        NUM_MPI_PROCS 1
+#        RUN_SERIAL
+#        )
+
+#      TRIBITS_ADD_EXECUTABLE_AND_TEST(
+#        FadKokkosAtomicTests_Cuda_Hierarchical_DFad
+#        SOURCES Fad_KokkosAtomicTests.hpp
+#                Fad_KokkosAtomicTests_Cuda_Hierarchical_DFad.cpp
+#        COMM serial mpi
+#        STANDARD_PASS_OUTPUT
+#        NUM_MPI_PROCS 1
+#        RUN_SERIAL
+#        )
+    ENDIF()
+
```

Comment:
Is commenting these out intentional or an oversight?  Were the tests ported to HIP?

Reply:
Intentional. I cut and pasted the CUDA block of unit tests so that we would have the same exact test coverage for HIP that we do for CUDA. These are the tests I could not get working - requires UVM or memory pool specializations. Until we sort out the memory pool and write a similar hip specilization I have left these disabled. I have enough working to proceed to panzer development and can come back to this after the Q2 deliverable is complete. These are the tests we talked about disabling late last week.